client/layout/index.jsx

```diff
@@ -167,6 +168,7 @@ Layout = React.createClass( {
 					isActive={ translator.isActivated() } />
 				{ this.renderPreview() }
 				{ config.isEnabled( 'happychat' ) && this.props.chatIsOpen && <AsyncLoad require="components/happychat" /> }
+				<AppBanner />
```

Comment:
Ah, I forgot about the notifications case. I'll let you know if I think of another work around.